src/shared/bus-unit-util.c

```diff
@@ -464,6 +464,22 @@ static int bus_append_cgroup_property(sd_bus_message *m, const char *field, cons
                 return 1;
         }
 
+        if (streq(field, "CPUQuotaPeriodSec")) {
+                usec_t u = 0;
```

Comment:
So I was thinking about this one in particular, and I'm not sure I agree in this case... I can see how using USEC_INFINITY makes sense for something like `WatchdogSec=` or `TimeoutSec=`, where setting it to infinity is essentially the same as disabling it... So I can see the logic in that.

But in this case, the default value is actually in the middle of the interval, so I'm not that convinced that 0 is not as good a choice as USEC_INFINITY would be.

In fact, I think someone setting `CPUQuotaPeriodSec=infinity` explicitly would expect to get the *maximum* allowed value in the interval (in other words, 1 second) rather than the default one (100 ms), no?